src/test/feature/ManagementTool/test_hawq_register.cpp

```diff
@@ -369,3 +369,25 @@ TEST_F(TestHawqRegister, TestUsage2Behavior2) {
   util.execute("drop table simple_register_table;");
   util.execute("drop table new_simple_register_table;");
 }
+
+TEST_F(TestHawqRegister, TestUsage2ForceOption) {
+  SQLUtility util;
+  util.execute("drop table if exists simple_register_table_force;");
+  util.execute("drop table if exists new_simple_register_table_force;");
+  util.execute("create table simple_register_table_force(i int) with (appendonly=true, orientation=row) distributed randomly;");
+  util.execute("insert into simple_register_table_force values(1), (2), (3);");
+  EXPECT_EQ(0, Command::getCommandStatus("hawq extract -d " + (string) HAWQ_DB + " -o tmp.yml testhawqregister_testusage2forceoption.simple_register_table_force"));
```

Comment:
I mean tmp_$testname.yml or similar. This helps users/developers to quickly
know what the file is for when something is wrong. This also avoids temp
file name conflicts when running cases in parallel in the future.

2016-09-04 21:25 GMT+08:00 Hong Wu <notifications@github.com>:

> In src/test/feature/ManagementTool/test_hawq_register.cpp
> <https://github.com/apache/incubator-hawq/pull/885#discussion_r77452500>:
>
> > @@ -369,3 +369,25 @@ TEST_F(TestHawqRegister, TestUsage2Behavior2) {
> >    util.execute("drop table simple_register_table;");
> >    util.execute("drop table new_simple_register_table;");
> >  }
> > +
> > +TEST_F(TestHawqRegister, TestUsage2ForceOption) {
> > +  SQLUtility util;
> > +  util.execute("drop table if exists simple_register_table_force;");
> > +  util.execute("drop table if exists new_simple_register_table_force;");
> > +  util.execute("create table simple_register_table_force(i int) with (appendonly=true, orientation=row) distributed randomly;");
> > +  util.execute("insert into simple_register_table_force values(1), (2), (3);");
> > +  EXPECT_EQ(0, Command::getCommandStatus("hawq extract -d " + (string) HAWQ_DB + " -o tmp.yml testhawqregister_testusage2forceoption.simple_register_table_force"));
>
> The meaningful name is tmp because of it is just a temporary yaml file
> without other meaning...
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-hawq/pull/885/files/b16f10b2b59f7d7f5bc81f789101c95af8716071#r77452500>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHI5jLp4m24by1aBj1Po_7MbjvDEc60hks5qmsa6gaJpZM4JzcWK>
> .
>